README.md

```diff
@@ -90,6 +90,41 @@ Please note that if you use a kubernetes cluster created on DigitalOcean, there
 will be a cloud controller manager running in the cluster already, so you local
 one will compete for API access with it.
 
+### Run Locally (optional parameters)
+
+In addition to the standard way of running `digitalocean-cloud-controller-manager`
+locally against a particular cluster, you can add some additional parameters that
+enable you to run the cluster with some extra features.
+
+If you want to include a `PUBLIC_ACCESS_FIREWALL_NAME` and `PUBLIC_ACCESS_FIREWALL_TAGS` for your cluster. You can run the following command.
+
+```bash
+cd cloud-controller-manager/cmd/digitalocean-cloud-controller-manager
+FAKE_REGION=fra1 DO_ACCESS_TOKEN=your_access_token PUBLIC_ACCESS_FIREWALL_NAME=firewall_name       \
+PUBLIC_ACCESS_FIREWALL_TAGS=k8s,k8s:8847f3c3-387f-4795-b794-b2f7d9f7d22f,k8s:worker go run main.go \
+  --kubeconfig <path to your kubeconfig file>                                                      \
+  --leader-elect=false --v=5 --cloud-provider=digitalocean
+```
+
+If you want to expose prometheus metrics, be sure to include the endpoint by
```

Comment:
I would create a subsection here named `Expose Prometheus Metrics` or something to that effect. 

Then you might write:

To expose a prometheus metrics endpoint:

```
... example ...
```

The `METRICS_ADDR` environment variable takes a colon-separated host:port address pair; both `host` and `port` parts are required.